make_openbazaar.sh

```diff
@@ -1,38 +1,34 @@
-﻿﻿#!/bin/sh
+#!/bin/bash
```

Comment:
Why not just use the system's shell. Are there things here that are Bash specific? And in most cases sh will resolve to bash anyways.